### PR TITLE
Scripts for environment setup / pipeline preparation

### PR DESCRIPTION
**Pull request by mattsmart**
todo better message here

### CODE REVIEW

File: models/tridimensional/docking_validation/config.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+DIR_PDB = os.path.join(DIR_BASE, "pdb")
+DIR_RESULTS = os.path.join(DIR_BASE, "results")
+DIR_SCRIPTS = os.path.join(DIR_BASE, "scripts")
+
+
+# ====================================================================== #
+# PDB:                                                                   #
+# ====================================================================== #
+#                                                                        #
+#     /- special -/    any special pdb files, esp. the original pdb      #
```
Comment (alexanian) [inline models/tridimensional/docking_validation/config.py:23]: We are pointing to the idea "any PDBs that we flag for extra testing or validation, e.g. VQR, EQR, the original". But "original" is only the WT PDB 4UN3.

File: models/tridimensional/docking_validation/config.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+FOLDERNAME_MUTANT_TEMPLATE = "template"
+FOLDERNAME_MUTANT_VARIANTS = "variants"
+MUTANT_TEMPLATE_PREFIX = "4UN3.mutant_"
+
+
+# ====================================================================== #
+# RESULTS:                                                               #
+# ====================================================================== #
+#                                                                        #
+#         /- batch -/   lone validation runs in labelled folders         #
```
Comment (alexanian) [inline models/tridimensional/docking_validation/config.py]: validation runs for a particular purpose/to check a particular things ("lone" confusing)

File: models/tridimensional/docking_validation/config.py [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+DIR_PDB_MUTANTS = os.path.join(DIR_BASE, "mutants")
+DIR_PDB_SPECIAL = os.path.join(DIR_BASE, "special")
+PATH_PDB_SPECIAL_ORIGINAL = os.path.join(DIR_PDB_SPECIAL, "4UN3.original.pdb")
+PATH_PDB_SPECIAL_VQR = os.path.join(DIR_PDB_SPECIAL, "4UN3.mutant_VQR.pdb")
+PATH_PDB_SPECIAL_EQR = os.path.join(DIR_PDB_SPECIAL, "4UN3.mutant_EQR.pdb")
+FOLDERNAME_MUTANT_TEMPLATE = "template"
+FOLDERNAME_MUTANT_VARIANTS = "variants"
+MUTANT_TEMPLATE_PREFIX = "4UN3.mutant_"
+
```
Comment (jrhawley) [inline models/tridimensional/docking_validation/config.py:50]: Add original pam variables for lengths 3, 4, and 5

File: models/tridimensional/docking_validation/config.py (hunk 2 above)
Comment (mattsmart) [inline models/tridimensional/docking_validation/config.py]: sure

**Comment by jrhawley** [comment]
Looks good, merging

**Pull request merged**